lib/forge2d_game.dart

```diff
@@ -33,8 +33,8 @@ class Forge2DGame extends BaseGame {
   }
 
   @override
-  void prepare(Component component) {
-    super.prepare(component);
+  Future<void> add(Component component) async {
+    super.add(component);
```

Comment:
Shouldn't we await for this before continuing?

Reply:
True, good catch! Fixed